hmftools.wdl

```diff
@@ -0,0 +1,927 @@
+        File gcProfile
+        
+        Int threads = 1
+        String memory = "5G"
+        String javaXmx = "4G"
+        Int timeMinutes = 240
+        String dockerImage = "quay.io/biocontainers/hmftools-cobalt:1.11--0"
+    }
+
+    command {
+        COBALT -Xmx~{javaXmx} \
+        -reference ~{normalName} \
+        -reference_bam ~{normalBam} \
+        -tumor ~{tumorName} \
+        -tumor_bam ~{tumorBam} \
+        -output_dir ~{outputDir} \
+        -threads ~{threads} \
+        -gc_profile ~{gcProfile}
+    }
+
+    output {
+        File version = "~{outputDir}/cobalt.version"
+        File normalGcMedianTsv = "~{outputDir}/~{normalName}.cobalt.gc.median.tsv"
+        File normalRationMedianTsv = "~{outputDir}/~{normalName}.cobalt.ratio.median.tsv"
+        File normalRationPcf = "~{outputDir}/~{normalName}.cobalt.ratio.pcf"
+        File tumorGcMedianTsv = "~{outputDir}/~{tumorName}.cobalt.gc.median.tsv"
+        File tumorRatioPcf = "~{outputDir}/~{tumorName}.cobalt.ratio.pcf"
+        File tumorRatioTsv = "~{outputDir}/~{tumorName}.cobalt.ratio.tsv"
+        File tumorChrLen = "~{outputDir}/~{tumorName}.chr.len"
+        Array[File] outputs = [version, normalGcMedianTsv, normalRationMedianTsv,
+            normalRationPcf, tumorGcMedianTsv, tumorRatioPcf, tumorRatioTsv, tumorChrLen]
+    }
+
+    runtime {
+        memory: memory
+        time_minutes: timeMinutes # !UnknownRuntimeKey
+        docker: dockerImage
+        cpu: threads
+    }
+
+    parameter_meta {
+        normalName: {description: "the name of the normal sample.", category: "required"}
+        normalBam: {description: "The normal BAM file.", category: "required"}
+        normalBamIndex: {description: "The index for the normal BAM file.", category: "required"}
+        tumorName: {description: "The name of the tumor sample.", category: "required"}
+        tumorBam: {description: "The tumor BAM file.", category: "required"}
+        tumorBamIndex: {description: "The index for the tumor BAM file.", category: "required"}
+        outputDir: {description: "The path to the output directory.", category: "common"}
+        gcProfile: {description: "A file describing the GC profile of the reference genome.", category: "required"}
+        threads: {description: "The number of threads the program will use.", category: "advanced"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        javaXmx: {description: "The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.",
+                  category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task Cuppa {
```

Comment:
No comment.

Reply:
I think this one is an abbreviation, Cancer of Unknown Primary